modules/apps/site-initializer/site-initializer-liferay-marketplace/src/main/resources/site-initializer/fragments/group/marketplace-base-fragments/fragments/get-app-button/index.html

```diff
@@ -0,0 +1,7 @@
+<div
+	class="d-none productID"
```

Comment:
Please, change the class name here.

```suggestion
	class="d-none product-id"
```